tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,272 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+from fauxfactory import gen_string
+from robottelo import (
+    manifests,
+    ssh,
+)
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from robottelo.cli.subscription import Subscription
+from robottelo.ssh import upload_file
+from robottelo.test import CLITestCase
+
+
+def line_count(self, file, connection=None):
+    """Get number of lines in a file."""
+    connection = connection or ssh.get_connection()
+    result = connection.run('wc -l < {0}'.format(file),
+                            output_format='plain')
+    self.assertEqual(result.return_code, 0, "BASH command error?")
+    return result
+
+
+class SimpleLoggingTestCase(CLITestCase):
+    """Test class for default logging to files."""
+
+    org = None
+    product = None
+
+    @classmethod
+    def setUpClass(cls):
+        """Tests for logging to files"""
+        super(SimpleLoggingTestCase, cls).setUpClass()
+
+        # need own org for the manifest refresh test
+
+        cls.org = make_org(cached=True)
+        cls.product = make_product_wait(
+            {u'organization-id': cls.org['id']},
+            )
+
+    def _make_repository(self, options=None):
+        """Makes a new repository and asserts its success"""
+        if options is None:
+            options = {}
+
+        if not options.get('product-id'):
+            options[u'product-id'] = self.product['id']
+
+        return make_repository(options)
+
+    def test_positive_logging_from_foreman_core(self):
```

Comment:
Could you please update the tier for all these tests ...

Reply:
OK, done. Thank you.